pages/index.js

```diff
@@ -1,122 +1,108 @@
-import Head from 'next/head'
-import ContactLinks from '../components/contactlinks'
-import Container from '../components/Navbar/Nav'
-import '../styles/Home.module.css'
+import Head from "next/head";
+import styles from "../styles/Home.module.css";
+import { faTwitter, faDribbble, faLinkedinIn, faKaggle, faMediumM, faGithub } from "@fortawesome/free-brands-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { useEffect } from "react";
+import ContactLinks from "./../components/contactlinks";
+import Container from "./../components/Navbar/Nav";
 
-var ReactRotatingText = require('react-rotating-text');
+var ReactRotatingText = require("react-rotating-text");
 export default function Home() {
+  useEffect(() => {
```

Comment:
If you have modified the education page only why have you made the changes in the index.js that is irrelevant then